Dockerfile

```
@@ -0,0 +1,14 @@
+FROM rust:alpine as builder
+RUN apk add clang musl-dev
+
+COPY . /tmp
+WORKDIR /tmp
+
+RUN cargo --version
+RUN cargo build --release --target x86_64-unknown-linux-musl
+
+FROM scratch as base
+COPY --from=builder /tmp/target/x86_64-unknown-linux-musl/release/convco /usr/bin/convco
```

if we remove the target, we have to update it here too.

```suggestion
COPY --from=builder /tmp/target/release/convco /usr/bin/convco
```